tests/globals/clusterers/minibatch_clusterer.py

```diff
@@ -1,14 +1,11 @@
 import pytest
-from relevanceai.http_client import Client, ClusterOps
+
+from relevanceai.http_client import Dataset
 
 from .utils import VECTOR_FIELDS
 
 
 @pytest.fixture(scope="session")
-def minibatch_clusterer(test_client: Client, vector_dataset_id: str):
-    df = test_client.Dataset(vector_dataset_id)
-    clusterer: ClusterOps = df.auto_cluster(
-        "minibatchkmeans-20", vector_fields=VECTOR_FIELDS
-    )
-    yield clusterer
-    clusterer.delete_centroids(df.dataset_id, VECTOR_FIELDS)
+def minibatch_clusterer(test_df: Dataset):
+    clusterer = test_df.auto_cluster("minibatchkmeans-20", vector_fields=VECTOR_FIELDS)
```

Comment:
Any reason to remove centroid deletion? I don't think it's redundant in case the API logic fails.

Reply:
you're right